src/PSRule.Rules.Azure/rules/Azure.APIM.Rule.ps1

```diff
@@ -266,6 +266,14 @@ Rule 'Azure.APIM.MinAPIVersion' -Ref 'AZR-000321' -Type 'Microsoft.ApiManagement
     }
 } -Configure @{ AZURE_APIM_MIN_API_VERSION = '2021-08-01' }
 
+# Synopsis: API Management instances should use multi-region deployment to reduce request latency perceived by geographically distributed API consumers.
+Rule 'Azure.APIM.MultiRegion' -Ref 'AZR-000340' -Type 'Microsoft.ApiManagement/service' -If { IsPremiumAPIM } -Tag @{ release = 'GA'; ruleSet = '2022_12'; } {
+    $multiRegion = @($TargetObject.properties.additionalLocations |
+        Where-Object { $_.location -and ($_.location -ne $TargetObject.location) -and ($_.sku.name -eq 'Premium') -and ($_.sku.capacity -ge 1) })
```

Comment:
It don't think we need add too much complexity here. Simplistically, if any additional locations are defined we meet the requirement. Although checking if the gateway is disabled is probably a good idea, we could also add this to another rule.

```suggestion
        Where-Object { $_.location -and ($_.location -ne $TargetObject.location) -and ($_.sku.name -eq 'Premium') -and ($_.sku.capacity -ge 1) })
```

Reply:
@BernieWhite 

Makes sense. By this, do you mean only check for `$_.location` and scratch -ge against primary region, SKU and capacity?

I think dedicated rule might be preferred for gateway and I can create that one as well :)